backend/server/application/response_draw.go

```diff
@@ -0,0 +1,33 @@
+package application
+
+import (
+	"chess/server/domain"
+)
+
+type ResponseDrawParam struct {
+	DrawResponse bool `json:"drawResponse"`
```

Comment:
Bool variable usually has a name that an '?' could be added at the end (DrawAccepted...?)

Reply:
I think this name is much more intuitive, however if you think otherwise I can change it